docs/manual/vsc-extension.md

```diff
@@ -55,11 +55,242 @@ We have also set some default keybindings you can use to interact with the lab w
+- **Position Nodes Geographically:** Use geographic coordinates (latitude and longitude) to place nodes on a world map.
+
+/// tab | Icon
+
+| Type     | Name         |
+| -------- | ------------ |
+| `string` | `graph-icon` |
+
+Using the `graph-icon` label you can define a role for a node (or a group of nodes if set under the kind/defaults section). Changing the role will change the icon of the node in the topology visualization.
+
+> When no icon label is defined. The node icon will default to `pe`.
+
+//// details | Available Roles and Icons
+    type: subtle-note
+
+| Role/Label name           | Icon                                           |
+| ------------------------- | ---------------------------------------------- |
+| **`pe`** / **`router`**   | ![](../images/topoviewer-icons/router.svg)     |
+| **`dcgw`**                | ![](../images/topoviewer-icons/dcgw.svg)       |
+| **`leaf`** / **`switch`** | ![](../images/topoviewer-icons/switch.svg)     |
+| **`spine`**               | ![](../images/topoviewer-icons/spine.svg)      |
+| **`server`**              | ![](../images/topoviewer-icons/server.svg)     |
+| **`pon`**                 | ![](../images/topoviewer-icons/pon.svg)        |
+| **`controller`**          | ![](../images/topoviewer-icons/controller.svg) |
+| **`rgw`**                 | ![](../images/topoviewer-icons/rgw.svg)        |
+| **`client`**              | ![](../images/topoviewer-icons/client.svg)     |
+
+////
+///
+/// tab | Group
+
+| Type     | Name          |
+| -------- | ------------- |
+| `string` | `graph-group` |
+
+Use `graph-group` label to categorize nodes into specific groups such as "Spine", "Leaf", etc.
+
+In the topology visualization this will group all nodes that belong to the group together and draw a box around them.
+
+///
+/// tab | Level
+
+| Type                 | Name          |
+| -------------------- | ------------- |
+| `integer (unsigned)` | `graph-level` |
+
+When you have groups defined, you can use the `graph-level` label to position the nodes hierarchically depending on their level value.
+
+The level value must be a positive integer.
+
+The `graph-level` value is defined in a reverse order, where `1` is the highest level.
+
+<h5>Vertical Layout</h5>
+
+When using the vertical layout algorithm to position nodes, the numerically higher the level value, the lower the group will be positioned relative to other groups.
+
+The group with the level value `1` will be positioned at the top of the visualization.
+
+<h5>Horizontal Layout</h5>
+
+When using the horizontal layout algorithm, the numerically higher the level value, the further left the group will be positioned relative to other groups.
+
+The group with the level value `1` will be positioned furthest to the left.
+
+///
+/// tab | Geo coordinates
+
+| Type     | Name                 |
+| -------- | -------------------- |
+| `string` | `geo-coordinate-lng` |
+| `string` | `geo-coordinate-lat` |
```

Comment:
@asadarafat I have removed aliases from the doc reference, as I think we should strive to have only one naming scheme.

Please check the long/lat names and adjust them in the provided sample topology if these ones are correct:

 ```
| `string` | `geo-coordinate-lng` |
| `string` | `geo-coordinate-lat` |
```